### ISSUE TITLE
Kint::$aliases fails for namespaced functions

### ISSUE THREAD

**Issue by Neunerlei** [kint-php/kint#316]
When I try to create a namespaced alias function like: 
```php
namespace Vendor\Dbg;
\Kint\Kint::$aliases[] = "Vendor\Dbg\dbg";
function dbg() {
	$args = func_get_args();
	Kint::dump(...$args);
}
``` 
Kint will remove the string from the list of aliases, because the regex (Utils.php:188) does not allow backslashes in a function name. To fix this issue it should be the same pattern as in (Utils.php:177). 

Thanks :)

**Comment by jnvsor** [comment]
Kint doesn't attempt to handle namespaces at all, mostly because it would have to take into account `use` statements, scopes, etc. etc. and this would vastly slow down parsing.

It should work if you just add `dbg` to the list of aliases.

However, I notice now that I have no test cases for namespaced functions, and I could probably simplify namespaced function names in `normalizeAliases` so I'll leave this open

**Comment by jnvsor** [comment]
Fixed in 47cca86694bbf97e5840f4e164c4513b1d22b671
